main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    guitar
+//  action 1:    strum the guitar
+guitar.strum();
+//  action 2:    tweak the guitar
+guitar.tweak();
+//  action 3:    tune the guitar
+guitar.tune();

```

Comment:
The guitar doesn't strum, tweak or tune.
someone else (a different object) strums, tweaks, or tunes the guitar. 
You have chosen actions that a 3rd party performs on the object. 
- [ ] revise.  Choose actions that THE OBJECT executes.